kf_lib_data_ingest/app/cli.py

```diff
@@ -262,6 +262,28 @@ def create_new_ingest(dest_dir=None):
     new_ingest_pkg(dest_dir)
 
 
+@click.command(name="validate")
+@click.argument(
+    "file_or_dir",
+    type=click.Path(exists=True, file_okay=True, dir_okay=True),
+)
+@click.option(
+    "--exclude_implicit",
+    default=False,
+    is_flag=True,
+)
+def validate_ingest(file_or_dir, exclude_implicit=False):
```

Comment:
Sorry gio, I meant to ask you before to change this back to `validate` so that it doesn't sound like its tied to ingest